### DISCUSSION TITLE
Maestral still under active development?

### DISCUSSION THREAD

**Discussion by ghost** [samschott/maestral#1023] [General]
Like the title says, is Maestral still under active development? I haven't noticed much movement in the issue queue by Sam, and there haven't been any new releases in a long time. @samschott

**Comment by samschott** [comment]
I would say it's more in maintenance mode. All essential feature are there and working, so expect to see a slower release cadence for bug fixes and compatibility with new Python releases.

**Comment by ghost** [comment]
Thanks for the update, Sam! 😃👍
